### ISSUE TITLE
Minimum PHP version.

### ISSUE THREAD

**Issue by Tanariel** [hiwye/paysafecard-php-api#1]
```
/**
     * Urls constructor
     *
     * @param array ...$urls
     */
    public function __construct(...$urls)
    {
        switch(sizeof($urls)){
            case 1:
                // success+failure+notification
                $this->setSuccessUrl($urls[0])->setFailureUrl($urls[0])->setNotificationUrl($urls[0]);
                break;
            case 2:
                // success+failure; notification
                $this->setSuccessUrl($urls[0])->setFailureUrl($urls[0])->setNotificationUrl($urls[1]);
                break;
            case 3:
                // success; failure; notification
                $this->setSuccessUrl($urls[0])->setFailureUrl($urls[1])->setNotificationUrl($urls[2]);
                break;
        }
    }
```

The '...' syntax in the constructor is only valid on php 5.6 or higher, can i suggest that you put a minimum php version requirement in the composer.json ?

**Comment by sebwalk** [comment]
Good catch @Tanariel! Thanks.
Do you want to make a PR or should I take care of it?

**Comment by Tanariel** [comment]
Sure :)

**Comment by sebwalk** [comment]
Merged #2 

Thanks!
